templates/santa/message.html.twig

```diff
@@ -105,6 +105,8 @@
                 </div>
             </div>
 
+            {% block time_limit %}{% endblock %}
```

Comment:
what about naming the block options instead - for consistency with the controller

Reply:
I agree, done